collectd_mesos_plugin.py

```diff
@@ -205,17 +206,21 @@ def run(self):
 
         # Get container inspect info and get marathon app and mesos task ids
         details = self._client.inspect_container(self._container['Id'])
-        app = None
-        task = None
+        app, task, kumo_job = None, None, None
         env = details.get('Config', {}).get('Env', [])
         for var in env:
-            name, value = var.split('=')
+            name, value = var.split('=')[:2]
```

Comment:
I think you are looking for something like: `name, _, value = var.partition('=')`  or `name, value = var.split('=', 1)`